docs/kubernetes/windows_demo.md

```diff
@@ -0,0 +1,227 @@
+    ```bash
+    PROJECT=${CLOUDSDK_CORE_PROJECT} \
+    NUM_NODES=2 \
+    NUM_WINDOWS_NODES=3 \
+    KUBE_GCE_ENABLE_IP_ALIASES=true \
+    KUBERNETES_NODE_PLATFORM=windows \
+    LOGGING_STACKDRIVER_RESOURCE_TYPES=new \
+    KUBERNETES_SKIP_CONFIRM=y \
+    go run ./hack/e2e.go -- --up
+    ```
+
+    **NOTE:** If the above command fails, try adding the flag `-get=false`.
+    ```bash
+    go run ./hack/e2e.go -get=false ...
+    ```
+
+    For further reference, [see](https://github.com/kubernetes/kubernetes/blob/master/cluster/gce/windows/README-GCE-Windows-kube-up.md).
+
+2. Untaint the Windows nodes. The easiest way to do this is running the smoke test:
+
+    ```bash
+    ./cluster/gce/windows/smoke-test.sh
+    ```
+
+3. From Filestore driver workspace, set up cluster.
+    ```bash
+    PROJECT=${CLOUDSDK_CORE_PROJECT} ./deploy/kubernetes/cluster_setup.sh
+    ```
+
+4. On one of the Windows nodes, create an SMB share.
+    
+    Remember the name of this node, will be needed next step.
+    ```Powershell
+    $Password = ConvertTo-SecureString "<your password>" -AsPlainText -Force
+    New-LocalUser -Name smbuser -AccountNeverExpires -Password $Password
+    Add-LocalGroupMember -Group "Remote Desktop Users" -Member smbuser
+    New-Item -ItemType "directory" -Path "C:\SMBShare"
+    New-SmbShare -Name "SMBShare" -Path "C:\SMBShare" -FullAccess "smbuser"
+    ```
+
+5. Update necessary YAML files in Filestore driver workspace.
+    
+    Information you need:
+    - Project name
+    - Name of the node that hosts the SMB share
+    - Password from previous step
+    - Name of the nodes you will be deploying to.
+
+    List of files:
+    - `deploy/kubernetes/manifests/node_windows.yaml`
+    - `examples/kubernetes/windows/secrets.yaml`
+    - `examples/kubernetes/windows/pv.yaml`
+    - `examples/kubernetes/windows/nettest-pod.yaml`
+    - `examples/kubernetes/windows/powershell-nettest-pod.yaml`
+
+## Deploying
+[//]: # (Use kubelet commands instead of logging in)
+1. Log in to the nodes you specified in `examples/kubernetes/windows/nettest-pod.yaml` and `examples/kubernetes/windows/powershell-nettest-pod.yaml`.
+2. Run the filestore driver.
+
+    The driver binary has to be run manually on Windows as there is no support for privileged containers. As a result, [CSI plugins don't work on Windows](https://github.com/kubernetes/enhancements/blob/master/keps/sig-windows/20190714-windows-csi-support.md). 
+
+    ```Powershell
+    $PROJECT = '<your project>'
+    cd C:\
+    gsutil cp gs://$PROJECT-bucket/gcfs-csi-driver.exe .
+    mkdir C:\var\lib\kubelet\plugins\filestore.csi.storage.gke.io
+    .\gcfs-csi-driver.exe --v=5 --endpoint="unix:/var/lib/kubelet/plugins/filestore.csi.storage.gke.io/csi.sock" --nodeid="(hostname)" --node=true
+    ```
+3. Back from the Filestore driver workspace, run the *daemonset* for the node-driver-registrar container.
+    ```bash
+    ./deploy/kubernetes/driver_start_windows.sh
```

Comment:
What's the command to check that this worked properly? Some get pods command with --namespace=something ...